core/backend/src/test/TestUtils.ts

```diff
@@ -57,6 +57,9 @@ export class TestUtils {
    * - cacheDir === path.join(__dirname, ".cache")
    */
   public static async startBackend(config?: IModelHostConfiguration): Promise<void> {
+    if (ProcessDetector.isIOSAppBackend) {
+      return IModelHost.startup();
```

Comment:
What was the need for this? Can we not directly set the cacheDir?

Reply:
@calebmshafer The cache dir is set correctly by default on native platforms, so we don't need to provide a path relative to __dirname. Doing so creates issues as the assets directory (the location of this webpacked code when it is executed) should not be written to by an iOS app at runtime.

Reply:
okay, I think even if we do skip setting the cacheDir we will still want to follow the configuration passed in. Can you try and see what happens when you just put an if-statement around line 64?

Reply:
@calebmshafer I moved the `isIOSBackend` check to line 64 so that we can make use of any provided configuration without setting the cacheDir.

Reply:
sounds good. thanks